go-controller/pkg/ovn/gateway_init.go

```diff
@@ -155,9 +158,34 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 		// router: UDP, TCP, SCTP
 		for _, proto := range enabledProtos {
 			if gatewayProtoLBMap[proto] == "" {
-				gatewayProtoLBMap[proto], stderr, err = util.RunOVNNbctl("--", "create",
+				if config.Gateway.Mode != config.GatewayModeLocal {
```

Comment:
Can you put the mode if as the external one? So it's more evident what balancers are created in which mode